contracts/Lens/PoolLens.sol

```diff
@@ -112,10 +113,10 @@ contract PoolLens is ExponentialNoError {
     struct RewardTokenState {
         // The market's last updated rewardTokenBorrowIndex or rewardTokenSupplyIndex
         uint224 index;
-        // The block number the index was last updated at
-        uint32 block;
-        // The block number at which to stop rewards
-        uint32 lastRewardingBlock;
+        // The block number or timestamp the index was last updated at
+        uint256 blockOrTimestamp;
+        // The block number or timestamp at which to stop rewards
+        uint256 lastRewardingBlockOrTimestamp;
```

Comment:
This seems a breaking change, maybe impacting the UI (cc @therealemjy)

Reply:
@therealemjy, As I can see that this struct is not externally used anywhere in the `PoolLens` contract, than how it will 
 impact the UI. This `RewardTokenState` Struct is neither being returned or given as an input in any function. Please LMK if I am missing something.

Reply:
Sorry, I assumed this was the struct used to return rewards. Please ignore my comment.